src/sentry/static/sentry/app/views/settings/project/projectServiceHooks.jsx

```diff
@@ -0,0 +1,161 @@
+import {Box} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import {Link} from 'react-router';
+import React from 'react';
+import createReactClass from 'create-react-class';
+
+import {t} from '../../../locale';
+import ApiMixin from '../../../mixins/apiMixin';
+import AsyncView from '../../asyncView';
+import {BooleanField} from '../../../components/forms';
+import Button from '../../../components/buttons/button';
+import EmptyMessage from '../components/emptyMessage';
+import IndicatorStore from '../../../stores/indicatorStore';
+import {Panel, PanelBody, PanelItem, PanelHeader} from '../../../components/panels';
+import SettingsPageHeader from '../components/settingsPageHeader';
+
+const ServiceHookRow = createReactClass({
+  displayName: 'ServiceHookRow',
+
+  propTypes: {
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    hook: PropTypes.object.isRequired,
+    onToggleActive: PropTypes.func.isRequired,
+  },
+
+  mixins: [ApiMixin],
+
+  getInitialState() {
+    return {
+      loading: false,
+      error: false,
+    };
+  },
+
+  render() {
+    let {orgId, projectId, hook} = this.props;
+    return (
+      <PanelItem>
+        <Box flex="1">
```

Comment:
I think you probably want a `<Field>` here too?

Reply:
O.o

Reply:
Can you explain what you mean? I dont understand why I'd want a field here. What I want is tabular data in a grid. There just happens to only be two columns.

Reply:
Also this is where I'd prefer to use the new BooleanField, but fitting a form in here is fairly illogical

Reply:
I guess there's no screenshot for this so I can't tell what it looks like, but if you used field it could give a more consistent design